code/rounds/InProgressRound.cs

```diff
@@ -75,11 +75,7 @@ protected override void OnStart()
                         player.SetHealth(player.MaxHealth);
                     }
 
-                    if (!player.IsForcedSpectator)
-                    {
-                        SetLoadout(player);
-                        _startingRoundPlayers.Add(player);
-                    }
+                    SetLoadout(player);
```

Comment:
I'd still give the loadout on spawn regarding death and the possibility searching other weapons which making it more obvious and leading to more weapons in game

Reply:
Hmm not sure if I fully understand, could you rephrase or PM me?

Previously we were setting the loadout twice which I want to avoid. The `SetLoadout(...)` call also happens right after we either respawn the player or set their hp to max.